lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -410,8 +410,8 @@ function dosomething_user_add_validation_attributes_to_address_fields($form, &$f
  *
  */
 function dosomething_user_validate_address_field($form, &$form_state) {
-  $first_name = $form_state['input']['field_first_name'][LANGUAGE_NONE][0]['value'];
-  $last_name = $form_state['input']['field_last_name'][LANGUAGE_NONE][0]['value'];
+  $first_name = $form_state['input']['field_first_name'][LANGUAGE_NONE][0]['value'];   
```

Comment:
gah, extra spaces. do you have the [`.editorconfig`](https://github.com/DoSomething/phoenix/blob/dev/.editorconfig) setup? these should automatically be trimmed